modules/platforms/cpp/ignite/client/ignite_client_configuration.h

```diff
@@ -187,6 +192,28 @@ class ignite_client_configuration {
         m_heartbeat_interval = heartbeat_interval;
     }
 
+    /**
+     * Gets the operation timeout, in milliseconds. Default is 0 (no timeout).
+     *
+     * An "operation" is a single client request to the server. Some public API calls may involve multiple operations,
+     * in which case the operation timeout is applied to each individual network call.
+     *
+     * @return Operation timeout, in milliseconds.
+     */
+    [[nodiscard]] std::chrono::milliseconds get_operation_timeout() const { return m_operation_timeout; }
+
+    /**
+     * Sets the operation timeout.
+     */
```

Comment:
Add some description of the parameter.
```suggestion
    /**
     * Sets the operation timeout.
     * 
     * @param operation_timeout Operation timeout.
     */
```